addOns/grpc/src/main/java/org/zaproxy/addon/grpc/ExtensionGrpc.java

```diff
@@ -22,18 +22,45 @@
 import org.parosproxy.paros.Constant;
 import org.parosproxy.paros.extension.ExtensionAdaptor;
 import org.parosproxy.paros.extension.ExtensionHook;
+import org.parosproxy.paros.view.View;
+import org.zaproxy.zap.view.ZapMenuItem;
 
 public class ExtensionGrpc extends ExtensionAdaptor {
```

Comment:
Please add tests verifying that the name and description of the extension is properly loaded (this would have caught the incorrect `Messages.properties` file).

Reply:
i have wrote a test but getting NullPointerExcerption error, i think Constant is not loading properly.

```
void checkNameDescExtension(){
        extensionGrpc = new ExtensionGrpc();
        assertEquals(Constant.messages.getString("grpc.name"), extensionGrpc.getUIName());
        assertEquals(Constant.messages.getString("grpc.desc"), extensionGrpc.getDescription());
    }
```

Reply:
@thc202 can you help me with test .

Reply:
See`ExtensionNetworkUnitTest`, key statements are extend of `TestUtils` and call of `mockMessages` (see also `shouldHaveUiName` which all is needed to assert the name is properly loaded).

Reply:
shouldHaveUiName is only checking the .getUIName is returning something or it should not be a empty string, my test is checking it for a particular string also which i think more appropriate.

Reply:
It's not an or, it must not be empty. I very doubt that the key is going to be changed ever again (see other extensions) that we need to validate it but I'm fine with that.